experimental/healthsea_spancat/scripts/evaluation.py

```diff
@@ -0,0 +1,356 @@
+
+    ner_fscore /= len(ner_scorer)
+    ner_recall /= len(ner_scorer)
+    ner_precision /= len(ner_scorer)
+
+    ner_data.append(
+        ("Average", round(ner_fscore, 2), round(ner_recall, 2), round(ner_precision, 2))
+    )
+
+    msg.divider("NER")
+    print(table(ner_data, header=header, divider=True))
+
+    print()
+
+    # Spancat Table
+    spancat_data = []
+    spancat_fscore = 0
+    spancat_recall = 0
+    spancat_precision = 0
+
+    for label in spancat_scorer:
+        spancat_data.append(
+            (
+                label,
+                round(spancat_scorer[label].fscore, 2),
+                round(spancat_scorer[label].recall, 2),
+                round(spancat_scorer[label].precision, 2),
+            )
+        )
+        spancat_fscore += spancat_scorer[label].fscore
+        spancat_recall += spancat_scorer[label].recall
+        spancat_precision += spancat_scorer[label].precision
+
+    spancat_fscore /= len(spancat_scorer)
+    spancat_recall /= len(spancat_scorer)
+    spancat_precision /= len(spancat_scorer)
+
+    spancat_data.append(
+        (
+            "Average",
+            round(spancat_fscore, 2),
+            round(spancat_recall, 2),
+            round(spancat_precision, 2),
+        )
+    )
+
+    msg.divider("Spancat")
+    print(table(spancat_data, header=header, divider=True))
+
+    print()
+
+    # KPI Table Config
+    kpi_header = ("Label", "Total Spans", "Correct NER", "Correct Spancat")
+    kpi_data = []
+
+    total_spans = 0
+    correct_ner_spans = 0
+    correct_spancat_spans = 0
+
+    for label in KPI:
+        kpi_data.append(
+            (
+                label,
+                KPI[label]["total_spans"],
+                f"{KPI[label]['correct_ner_spans']} ({round((KPI[label]['correct_ner_spans']/KPI[label]['total_spans'])*100,2)}%)",
+                f"{KPI[label]['correct_spancat_spans']} ({round((KPI[label]['correct_spancat_spans']/KPI[label]['total_spans'])*100,2)}%)",
+            )
+        )
+
+        total_spans += KPI[label]["total_spans"]
+        correct_ner_spans += KPI[label]["correct_ner_spans"]
+        correct_spancat_spans += KPI[label]["correct_spancat_spans"]
+
+    kpi_data.append(
+        (
+            "Total",
+            total_spans,
+            f"{correct_ner_spans} ({round((correct_ner_spans/total_spans)*100,2)}%)",
+            f"{KPI[label]['correct_spancat_spans']} ({round((correct_spancat_spans/total_spans)*100,2)}%)",
+        )
+    )
+
+    msg.divider("KPI")
+    print(table(kpi_data, header=kpi_header, divider=True))
+
+    coverage = round((matching_candidates / total_real_candidates) * 100, 2)
+    candidates_relation = round((total_candidates / total_real_candidates) * 100, 2)
+
+    msg.divider("Suggester KPI")
+
+    if "candidates_indices" not in spancat_doc.user_data:
+        msg.info("save_candidates is not set to true")
+    else:
+        msg.info("save_candidates is set to true")
```

Comment:
How do I set these to `True`? :-)

Reply:
I've deleted this part, since this is directly related to the `set_candidates` [PR](https://github.com/explosion/spaCy/pull/10413) and it's not that important here 😄